internal/kafka/internal/api/public/api/openapi.yaml

```diff
@@ -1106,7 +1106,7 @@ components:
         updated_at: 2020-10-05T12:56:36.362208Z
         version: 2.6.0
         instance_type: standard
-        browser_url: https://console.redhat.com/
+        browser_url: http://localhost:8080/
```

Comment:
Why this ?

Reply:
I believe it was because of this discussion https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/838#discussion_r826056113

Reply:
Yes! Good suggestion by @JameelB 
Using default value for development purposes is ok, but OpenAPI is user facing so if possible can we leave examples and others with still console.redhat.com while default in the code will be localhost (and we can override that in deployment)

Reply:
This is just suggestion. Ignore me completely if that doesn't make sense :D